sdk/communication/test-resources.json

```diff
@@ -50,6 +50,22 @@
         "type": "string",
         "value": "[listKeys(resourceId('Microsoft.Communication/CommunicationServices',variables('uniqueSubDomainName')), '2020-08-20-preview').primaryConnectionString]"
       },
+      "ADMINISTRATION_SERVICE_ENDPOINT": {
+        "type": "string",
+        "value": "[concat('https://', parameters('baseName'), '-', parameters('endpointPrefix'), '.communication.azure.com')]"
+      },
+      "ADMINISTRATION_SERVICE_ACCESS_TOKEN": {
+        "type": "string",
+        "value": "[listKeys(resourceId('Microsoft.Communication/CommunicationServices',variables('uniqueSubDomainName')), '2020-08-20-preview').primaryKey]"
+      },
+      "CHAT_SERVICE_ENDPOINT": {
```

Comment:
We should probably consolidate that with the COMMUNICATION_SERVICES_CONNECTION_STRING in the future. In JS, Chat also uses their own separate env variable.

Reply:
Agreed, in Java, each package uses its own separate env variable, I think this can all be consolidated in the future.